ovs/lib/vdisk.py

```diff
@@ -608,6 +608,38 @@ def delete_snapshots(snapshot_mapping):
                     results[vdisk_guid] = [False, msg]
                 continue
 
+            VDiskController._delete_multiple_snapshots(vdisk, snapshot_ids, results)
+
+            vdisk.invalidate_dynamics(['snapshots', 'snapshot_ids'])
+            if backwards_compat is True:
+                results[vdisk_guid] = results[vdisk_guid]['results'][snapshot_ids[0]]
+        return results
+
+    @staticmethod
+    def delete_snapshots_concurrent(vdisk, snapshots=None, skip_used_snapshots=True):
+        """
+        Delete vDisk snapshots in a range, making use of the volumedriver concurrent call
+        :param vdisk: vdisk to delete snapshots from via the storagedriver client
+        :type vdisk: ovs.dal.hybrids.vdisk
+        :param snapshots: list of snapshot ids to remove
+        :type snapshots: list
+        :param skip_used_snapshots: skip deletion of snapshots if they are in use
+        :type skip_used_snapshots: bool
+        :return: the volumedriver std_client output value
+        :rtype: any
+        """
+        if not snapshots:
+            snapshots = vdisk.snapshot_ids
+        return vdisk.storagedriver_client.delete_snapshots(volume_id=str(vdisk.volume_id),
+                                                           snapshots=snapshots,
+                                                           skip_used_snapshots=skip_used_snapshots)
+        #todo this does not output anything. should be fixed after volumedriver-ee/issues/222 is fixed and make the output uniformous with _delete_multiple_snapshots
+
+    @staticmethod
+    def _delete_multiple_snapshots(vdisk, snapshot_ids, results):
```

Comment:
Docstrings
Also why pass in the results dict? Just return a similar format so the function calling it can update its own results